src/briefcase/commands/run.py

```diff
@@ -218,13 +225,90 @@ def add_options(self, parser):
         self._add_update_options(parser, context_label=" before running")
         self._add_test_options(parser, context_label="Run")
 
-    def _prepare_app_kwargs(self, app: AppConfig):
+        if self.supports_debugger:
+            self._add_debug_options(parser, context_label="Run")
+            parser.add_argument(
+                "--debugger-host",
+                default="localhost",
+                help="The host on which to run the debug server (default: localhost)",
+                required=False,
+            )
+            parser.add_argument(
+                "-dp",
+                "--debugger-port",
+                default=5678,
+                type=int,
+                help="The port on which to run the debug server (default: 5678)",
+                required=False,
+            )
```

Comment:
Is there a reason these options aren't part of `_add_debug_options()`?

If it's to avoid having the ports shown for the `build` command - should we include a `build=False` argument so that any run command that needs the options can add them?